### PR TITLE
Aria accordion

### PR DESCRIPTION
**Pull request by salamaashoush**
Reimplement Accordion component using aria hooks

### CODE REVIEW

File: packages/libs/react-ui/src/components/Accordion/Accordion.css.ts [hunk 1]
```diff
@@ -1,70 +1,60 @@
...
-    borderBottom: `1px solid ${vars.colors.$borderDefault}`,
-    selectors: {
-      '&:last-child': {
-        marginBottom: 0,
-      },
+export const accordionSectionClass = style({
+  display: 'block',
+  marginBlockEnd: token('spacing.md'),
+  overflow: 'hidden',
+  borderBlockEnd: `1px solid ${token('color.border.base.default')}`,
```
Comment (eileenmguo) [inline packages/libs/react-ui/src/components/Accordion/Accordion.css.ts:8]: you can implement these with atoms

File: packages/libs/react-ui/src/components/Accordion/Accordion.stories.tsx [hunk 2]
```diff
@@ -45,47 +23,33 @@ const meta: Meta<StoryProps> = {
...
-                key={index}
-              >
-                {children}
-              </Accordion.Section>
-            ),
-          )}
-        </Accordion.Root>
-      </div>
+      <Accordion selectionMode={selectionMode}>
+        <Item key="files" title="Your files">
```
Comment (eileenmguo) [inline packages/libs/react-ui/src/components/Accordion/Accordion.stories.tsx:43]: Should we use AccordionItem for consistency?

File: packages/libs/react-ui/src/components/Icon/IconWrapper.tsx [hunk 3]
```diff
@@ -15,9 +15,10 @@ export const IconWrapper = (
 ): React.FC<IconType> => {
   const WrappedIcon: React.FC<SVGProps<SVGSVGElement> & IIconProps> = ({
     size = 'md',
+    className,
     ...props
   }) => (
-    <span className={classNames(iconContainer, sizeVariants[size])}>
+    <span className={classNames(iconContainer, sizeVariants[size], className)}>
```
Comment (eileenmguo) [inline packages/libs/react-ui/src/components/Icon/IconWrapper.tsx:21]: 👍

**Review by eileenmguo — COMMENTED** [review]
The icon transition looks strange and the expanding animation is no longer there

https://github.com/kadena-community/kadena.js/assets/9022549/4b5e7865-824d-4cab-9e83-b280523da45d

**Review by alber70g — APPROVED** [review]

**Comment by alber70g** [comment]
I approved for codeowners files only.

File: packages/apps/tools/src/pages/home/index.tsx [hunk 4]
```diff
@@ -229,13 +230,13 @@ const Home: FC = () => {
               {t('Frequently Asked Questions')}
             </Heading>
             <Box marginBlockEnd="md" />
-            <Accordion.Root>
-              {faqs.map((faq) => (
-                <Accordion.Section title={faq.title} key={faq.title}>
+            <Accordion items={faqs}>
+              {(faq) => (
+                <AccordionItem title={faq.title} key={faq.title}>
```
Comment (sanderlooijenga) [inline packages/apps/tools/src/pages/home/index.tsx:235]: Before (on current tools.kadena.io ) you could expand both items. Is this no longer possible?

**Review by sanderlooijenga — APPROVED** [review]

File: packages/apps/tools/src/pages/home/index.tsx (hunk 4 above)
Comment (salamaashoush) [reply packages/apps/tools/src/pages/home/index.tsx:235]: you can do it, it was not clear that this is the case, the new Accordion has `selectionMode` by default it is `single` but possible values are `multiple` and `none`, I will change this to `multiple` if that is the required behavior

**Review by eileenmguo — APPROVED** [review]

**Pull request merged**